service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -359,6 +372,105 @@ public OrderWithAddressesResponseDto saveCurrentAddressForOrder(OrderAddressDtoR
         return findAllAddressesForCurrentOrder(uuid);
     }
 
+    private OrderAddressDtoRequest geoCodeSearchingRequest(String searchRequest) {
```

Comment:
it's better to implement this google-related functionality in a separate service.